### ISSUE TITLE
Docker infrastructure is broken and packaging has some issues

### ISSUE THREAD

**Issue by snak1219** [faucetsdn/chewie#169]
I would make a separate pull request (dealing with at least the docker stuff) but I would like some clarifications first if possible. I believe right now chewie won't run from the current setup because either the packaging or something to do with the  sys path has some issues.

If you clone the repo and try to run __main__.py, it will throw this error:
![chewieerror](https://user-images.githubusercontent.com/22568590/62746971-161c4680-ba07-11e9-8173-2f5416e1f3fa.png)

Doing docker compose --build or creating the image using the below command seen in the documentation results with the same problem.
 
`docker build -t chewie_image -f Dockerfile.chewie .`

Now to fix this temporarily on my machine, I've added the below line into my __main__.py file.

`sys.path.insert(0, "/home/snakamura/code/chewie")`

Once I do this I can run it natively fine from *only* that directory. 

![workingchewie](https://user-images.githubusercontent.com/22568590/62747010-47951200-ba07-11e9-8e9a-282968200faf.png)

Clearly this is a terrible temp fix so it would be really great if I could have some guidance on fixing this issue. Thank you!

**Comment by snak1219** [comment]
Also if you do a docker-compose or even build the image and mount the source code, it won't actually start the chewie container because of this issue.

**Comment by gizmoguy** [comment]
Chewie needs to be run as a module instead of just as a standalone python file (otherwise you get python path errors like you're getting).

I see there are lots of other problems with the dockerfiles, docker-compose.yml and the documentation on running the dockers. I'll loop back next week and fix these.

For now I've pushed up a basic branch here that fixes the Dockerfile for chewie: https://github.com/faucetsdn/chewie/pull/170

Want to give that a go and see how it works?

```
$ sudo docker build -t faucet/chewie -f Dockerfile.chewie .
Sending build context to Docker daemon  2.109MB
....
Successfully built be1416fe5c2b
Successfully tagged faucet/chewie:latest
```

```
$ sudo docker run -it --rm faucet/chewie
Starting with UID=0 GID=0
2019-08-09 05:27:33,155 - CHEWIE - INFO - Starting Chewie...
2019-08-09 05:27:33,155 - CHEWIE.Chewie - INFO - Starting
2019-08-09 05:27:33,193 - CHEWIE.Chewie - INFO - Radius Listening on 0.0.0.0:0
2019-08-09 05:27:33,194 - CHEWIE.Chewie - INFO - waiting for eap.
2019-08-09 05:27:33,194 - CHEWIE.Chewie - INFO - waiting for ip activity.
2019-08-09 05:27:33,194 - CHEWIE.Chewie - INFO - waiting for radius.
```

**Comment by snak1219** [comment]
Thank you! I’ll check it out tomorrow!

**Comment by snak1219** [comment]
Thank your patch works for me as well. Now if I want to deploy a 802.1x environement with faucet for a isolated test environment, do you recommend that I use hostapd or chewie, or some other authenticator at this stage?

**Comment by gizmoguy** [comment]
Hi @snak1219, we definitely recommend chewie over hostapd, earlier versions of faucet 802.1x used hostapd and we found their event API incredibly frustrating so we wrote chewie as a replacement for hostapd.

**Comment by snak1219** [comment]
@gizmoguy thank you!!!

**Comment by gizmoguy** [comment]
Full fix implemented in #175
